docs/API.md

```diff
@@ -137,7 +137,7 @@ connection to a specified VNC server.
 
 ##### Syntax
 
-    var rfb = new RFB( target, url [, options] );
+    const rfb = new RFB( target, url [, options] );
```

Comment:
Bike shedding, but I'd say `let` is more general and should be in the examples.

Reply:
It's a common practice nowadays to use const whenever possible.
It makes code more solid since you know that the variable won't change since it's declaration.
Also helps to get more "functional" code.

but yeah. It's trivial... I can change to let if you prefer that :)

Reply:
Right. You just need to separate the case where it was truly meant to never change, and when the code just happened to end up that way. There is some intent implied when you use `const`.

I don't have any really strong emotions about this particular case. I was just thinking about how we use it in `ui.js` where it is very much not a constant.